### PR TITLE
Init template

### PR DESCRIPTION
**Pull request by Mihailmihail111**

### CODE REVIEW

**Review by Kravalg — CHANGES_REQUESTED** [review]
Also please add CI checks through Github Actions for pull requests (psalm, phpunit, behat , [symfony security check](https://symfony.com/doc/current/setup.html#checking-security-vulnerabilities), composer validate, [rips security checker](https://github.com/robocoder/rips-scanner), [sonar security checker](https://github.com/SonarSource/sonar-php))

Also add changelog generation and publish the first version v1.0.0 of this repository

**Comment by vilnacrm** [comment]
add @Kravalg and @Mihailmihail111 to codeowners
https://stackoverflow.com/questions/46340474/set-a-default-reviewer

File: README.md [hunk 1]
```diff
@@ -1 +1,31 @@
-# php-service-template
```
[the remark this answers is not in the document]
Comment (Kravalg) [reply README.md:1]: fixed

**Comment by Kravalg** [comment]
the wrong pr

**Pull request closed without merging**